promgen/forms.py

```diff
@@ -331,3 +332,29 @@ def __init__(self, *args, **kwargs):
         if input_object:
             self.fields["permission"].choices = get_permission_choices(input_object)
         self.fields["users"].choices = get_user_choices()
+
+
+class UserMergeForm(forms.Form):
+    user_to_merge_from = forms.ChoiceField(required=True)
+    user_to_merge_into = forms.ChoiceField(required=True)
+
+    def __init__(self, *args, **kwargs):
+        super(UserMergeForm, self).__init__(*args, **kwargs)
+        self.fields["user_to_merge_from"].choices = get_user_choices()
+        self.fields["user_to_merge_from"].label = _(
+            "User to merge from (This account will be deleted)"
+        )
+        self.fields["user_to_merge_into"].choices = get_user_choices()
+        self.fields["user_to_merge_into"].label = _(
+            "User to merge into (This account will be kept)"
+        )
+
+    def clean(self):
+        cleaned_data = super().clean()
+        user_to_merge_from = cleaned_data.get("user_to_merge_from")
+        user_to_merge_into = cleaned_data.get("user_to_merge_into")
+
+        if user_to_merge_from == user_to_merge_into:
+            raise ValidationError(_("User_to_merge_from and User_to_merge_into must be different."))
```

Comment:
It seems that this is a message in natural language, as I can see by the translation you have added for it:

```
#: promgen/forms.py:359
msgid "User_to_merge_from and User_to_merge_into must be different."
msgstr "統合元アカウントと統合先アカウントは、別のアカウントである必要があります。"
```

Therefore you should write "User to merge from and user to merge into must be different", instead of "User_to_merge_from ..." which seems to refer to a variable name, although is not exactly the variable name because it starts with capital U. In the Japanese translation there isn't any reference to the variable name, therefore I understand that sentence is written in natural language.